### PR TITLE
Allow for http method selection in custom webhook

### PR DESCRIPTION
**Pull request by jcleezer**
*Issue #, if available:* None

*Description of changes:*
Adds a dropdown to the custom webhook construction page that allows the user to choose the HTTP method to use.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: public/pages/Destinations/components/createDestinations/CustomWebhook/MethodEditor.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { FieldArray } from 'formik';
+import SubHeader from '../../../../../components/SubHeader';
+import { FormikSelect } from '../../../../../components/FormControls';
+
+
+const methodOptions = [{ value: 'POST', text: 'POST' }, { value: 'PUT', text: 'PUT' }, { value: 'PATCH', text: 'PATCH' }, { value: 'GET', text: 'GET' }, { value: 'DELETE', text: 'DELETE' }];
```
Comment (skkosuri-amzn) [inline public/pages/Destinations/components/createDestinations/CustomWebhook/MethodEditor.js:8]: Remove `DELETE` and `GET`, as we deleted those from the backend changes

Comment (jcleezer) [reply public/pages/Destinations/components/createDestinations/CustomWebhook/MethodEditor.js]: Thanks! Removed.

**Comment by skkosuri-amzn** [comment]
@ftianli-amzn : could you please check this PR.

File: public/pages/Destinations/components/createDestinations/CustomWebhook/MethodEditor.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import { FieldArray } from 'formik';
+import SubHeader from '../../../../../components/SubHeader';
+import { FormikSelect } from '../../../../../components/FormControls';
+
+
+const methodOptions = [{ value: 'POST', text: 'POST' }, { value: 'PUT', text: 'PUT' }, { value: 'PATCH', text: 'PATCH' }];
+
+
+
+const propTypes = {
```
Comment (ftianli-amzn) [inline public/pages/Destinations/components/createDestinations/CustomWebhook/MethodEditor.js:12]: Is the type checking effective here?
I thought the usage is either `MethodEditor.propTypes = { ... };`, or adding an additional line `MethodEditor.propTypes = propTypes;` besides the `const propTypes = { ...}`

Comment (jcleezer) [reply public/pages/Destinations/components/createDestinations/CustomWebhook/MethodEditor.js:12]: Thanks! Went with `MethodEditor.propTypes = propTypes;` to be consistent with the other editors.

**Review by ftianli-amzn — APPROVED** [review]

**Review by skkosuri-amzn — APPROVED** [review]

**Pull request merged**